etc/html/HELP.html

```diff
@@ -152,7 +152,7 @@ <h3><a id="class_mem"></a>List of Data and Function Members</h3>
 them into a file.
 Because methods should have reasonable names, often the method name itself is already
 a hint on what it does.
-If you want to get a <a href="http://root.cern.ch/root/html/TNamed.html">TNamed</a>'s
+If you want to get a <a href="http://root.cern/root/html/TNamed.html">TNamed</a>'s
```

Comment:
Link should be replaced by https://root.cern/doc/master/classTNamed.html

Reply:
http://root.cern/root/html/TNamed.html Also works